src/main/java/net/fabricmc/meta/utils/Reference.java

```diff
@@ -37,4 +38,6 @@ public final class Reference {
 	 * Url to the MC metadata json listing all the game versions.
 	 */
 	public static final String MC_METADATA_URL = "https://piston-meta.mojang.com/mc/game/version_manifest.json";
+
+	public static final MavenVersion NOOP_INTERMEDIARY_VERSION = new MavenVersion("net.fabricmc:intermediary:0.0.0", true);
```

Comment:
I dont think this is going to work for everything, I believe a lot of launchers assume that there will be a `net.fabricmc:intermediary:<mc_version>` for the given mc version.